earthpy/clip.py

```diff
@@ -30,13 +29,11 @@ def _clip_points(shp, clip_obj):
 
 
 def _clip_line_poly(shp, clip_obj):
-    """A function to clip line and polygon data using geopandas.
+    """Clip line and polygon data using geopandas.
 
```

Comment:
Actually i'm wondering if we need the longer docstring in this instance? the short one should be enough i think? @mbjoseph what do you think about these docstrings? keep is short and sweet? i'm not sure that people need the detail of us using spatial indices and such.